src/containers/Subscribe/index.js

```diff
@@ -22,11 +22,14 @@ export class Subscribe extends React.Component {
         notificationStatus: subscriptionStatus,
       });
       if (message.length !== 0) {
+        Materialize.Toast.dismissAll();
         Materialize.toast({ html: message });
       }
       if (errors === 'Network Error') {
+        Materialize.Toast.dismissAll();
```

Comment:
Add test for network errors and check if the toast is dismissed

Reply:
I have refactored this part by removing this block because there is a story for handling all errors that @mutaimwiti  and @YAHYA-H  are working on and they have a PR